src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -141,7 +142,7 @@ def sgn(x, y):
             l1j = l1[j]
             for k in range(j):
                 if l2j < l2[k]:
-                    t = (l1j[0] - l1[k][0]) / ((l2[k][0] - l2j[0]) + (l1j[0] - l1[k][0]))
+                    t = (l1j[0]-l1[k][0])/((l2[k][0] - l2j[0]) + (l1j[0]-l1[k][0]))
```

Comment:
Personally, I find the changed code less readable; specially around the `/`. However, I just wanted to mention this. If you're happy with it, then no need to (partially) revert.